misk-testing/src/test/kotlin/misk/logging/LogCollectorTest.kt

```diff
@@ -14,8 +15,9 @@ import kotlin.test.assertFailsWith
 
 @MiskTest(startService = true)
 class LogCollectorTest {
```

Comment:
I don't particularly enjoy that this test appears to be a copy of the wisp LogCollectorTest, but 🤷  I added the new test here too.